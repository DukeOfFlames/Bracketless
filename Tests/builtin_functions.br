START

#drucke{#max{{4, 6, 8}}}

°list = {4, 6, 8}
°max = #max{list}
°hex = #hex{13}
°bin = #bin{4!}
#drucke{max}
#drucke{hex}
#drucke{bin}

§ °list = #max{{4, 6, 8}}

END