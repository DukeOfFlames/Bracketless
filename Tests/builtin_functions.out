8
8
0xd
0b11000
0.6
